src/phoenix/server/api/input_types/PatchAnnotationsInput.py

```diff
@@ -0,0 +1,17 @@
+from typing import Optional
+
+import strawberry
+from strawberry import UNSET
+from strawberry.relay import GlobalID
+from strawberry.scalars import JSON
+
+
+@strawberry.input
+class PatchAnnotationsInput:
```

Comment:
Suggestion: I might name this something like `AnnotationPatch` since it is not actually the input to the patch annotations resolver, but is just one element of a list.